packages/fxa-auth-server/lib/metrics/amplitude.js

```diff
@@ -17,30 +17,33 @@ const { version: VERSION } = require('../../package.json');
 
 // Maps template name to email type
 const EMAIL_TYPES = {
-  lowRecoveryCodesEmail: '2fa',
-  newDeviceLoginEmail: 'login',
-  passwordChangedEmail: 'change_password',
-  passwordResetEmail: 'reset_password',
-  passwordResetAccountRecoveryEmail: 'account_recovery',
-  passwordResetRequiredEmail: 'reset_password',
-  postChangePrimaryEmail: 'change_email',
-  postRemoveSecondaryEmail: 'secondary_email',
-  postVerifyEmail: 'registration',
-  postVerifySecondaryEmail: 'secondary_email',
-  postAddTwoStepAuthenticationEmail: '2fa',
-  postRemoveTwoStepAuthenticationEmail: '2fa',
-  postAddAccountRecoveryEmail: 'account_recovery',
-  postRemoveAccountRecoveryEmail: 'account_recovery',
-  postConsumeRecoveryCodeEmail: '2fa',
-  postNewRecoveryCodesEmail: '2fa',
-  recoveryEmail: 'reset_password',
+  downloadSubscription: 'subscription_download',
```

Comment:
After adding tests, noticed that `downloadSubscription`, `verificationReminderFirst`, `verificationReminderSecond` were not being reported to amplitude, so adding here fixes that issue.